src/constants/exchangeConstants.js

```diff
@@ -17,45 +17,18 @@
     with this program; if not, write to the Free Software Foundation, Inc.,
     51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
-export const ADD_OFFER = 'ADD_OFFER';
-export const RESET_OFFERS = 'RESET_OFFERS';
-export const SET_EXCHANGE_SEARCH_REQUEST = 'SET_EXCHANGE_SEARCH_REQUEST';
-export const SET_EXECUTING_TRANSACTION = 'SET_EXECUTING_TRANSACTION';
-export const SET_DISMISS_TRANSACTION = 'SET_DISMISS_TRANSACTION';
-export const SET_EXCHANGE_ALLOWANCES = 'SET_EXCHANGE_ALLOWANCES';
-export const ADD_ACCOUNT_EXCHANGE_ALLOWANCE = 'ADD_ACCOUNT_EXCHANGE_ALLOWANCE';
-export const UPDATE_ACCOUNT_EXCHANGE_ALLOWANCE = 'UPDATE_ACCOUNT_EXCHANGE_ALLOWANCE';
-export const SET_EXCHANGE_SUPPORTED_ASSETS = 'SET_EXCHANGE_SUPPORTED_ASSETS';
-export const SET_FIAT_EXCHANGE_SUPPORTED_ASSETS = 'SET_FIAT_EXCHANGE_SUPPORTED_ASSETS';
-export const SET_WBTC_PENDING_TRANSACTIONS = 'SET_WBTC_PENDING_TRANSACTIONS';
-export const SET_WBTC_SETTLED_TRANSACTIONS = 'SET_WBTC_SETTLED_TRANSACTIONS';
-export const SET_UNISWAP_TOKENS_QUERY_STATUS: 'SET_UNISWAP_TOKENS_QUERY_STATUS' = 'SET_UNISWAP_TOKENS_QUERY_STATUS';
 
-export const PROVIDER_UNISWAP = 'UNISWAPV2-SHIM';
-export const PROVIDER_1INCH = 'ONEINCH-SHIM';
-export const PROVIDER_SYNTHETIX = 'SYNTHETIX';
-
-export const MARK_NOTIFICATION_SEEN = 'MARK_NOTIFICATION_SEEN';
-
-export const EXCHANGE = 'EXCHANGE';
+// Note: these values are matching Etherspot values
```

Comment:
Is it possible maybe to export constants from Etherspot SDK in case it changes?

Reply:
Resolved by separating these types & creating `parseExchangeProvider` function. Now app should continue working even if etherspot sends new/unknown provider.